src/provider/amino/types/gov.ts

```diff
@@ -311,70 +311,73 @@ const checkEncodeProposal = (content: any, amino: AminoValueMap): DirectProposal
         },
       };
     }
-    case ContentTypes[GovUtils.ProposalTypes.SetMsgGasCost]: {
-      const setMsgGasCostMsg = preProcessAmino(content.value.msg, {});
-      const setMsgGasCostProp = Carbon.Fee.SetMsgGasCostProposal.fromPartial({
-        ...content.value,
-        msg: setMsgGasCostMsg,
-      });
-      return {
-        newContent: {
-          typeUrl: GovUtils.ProposalTypes.SetMsgGasCost,
-          value: Carbon.Fee.SetMsgGasCostProposal.encode(setMsgGasCostProp).finish(),
-        },
-        newAmino: {
-          ...amino,
-        },
-      };
-    }
-    case ContentTypes[GovUtils.ProposalTypes.SetMinGasPrice]: {
-      const setMinGasPriceMsg = preProcessAmino(content.value.msg, {});
-      const setMinGasPriceProp = Carbon.Fee.SetMinGasPriceProposal.fromPartial({
-        ...content.value,
-        msg: setMinGasPriceMsg,
-      });
-      return {
-        newContent: {
-          typeUrl: GovUtils.ProposalTypes.SetMinGasPrice,
-          value: Carbon.Fee.SetMinGasPriceProposal.encode(setMinGasPriceProp).finish(),
-        },
-        newAmino: {
-          ...amino,
-        },
-      };
-    }
-    case ContentTypes[GovUtils.ProposalTypes.RemoveMsgGasCost]: {
-      const removeMsgGasCostMsg = preProcessAmino(content.value.msg, {});
-      const removeMsgGasCostProp = Carbon.Fee.RemoveMsgGasCostProposal.fromPartial({
-        ...content.value,
-        msg: removeMsgGasCostMsg,
-      });
-      return {
-        newContent: {
-          typeUrl: GovUtils.ProposalTypes.RemoveMsgGasCost,
-          value: Carbon.Fee.RemoveMsgGasCostProposal.encode(removeMsgGasCostProp).finish(),
-        },
-        newAmino: {
-          ...amino,
-        },
-      };
-    }
-    case ContentTypes[GovUtils.ProposalTypes.RemoveMinGasPrice]: {
-      const removeMinGasPriceMsg = preProcessAmino(content.value.msg, {});
-      const removeMinGasPriceProp = Carbon.Fee.RemoveMinGasPriceProposal.fromPartial({
-        ...content.value,
-        msg: removeMinGasPriceMsg,
-      });
-      return {
-        newContent: {
-          typeUrl: GovUtils.ProposalTypes.RemoveMinGasPrice,
-          value: Carbon.Fee.RemoveMinGasPriceProposal.encode(removeMinGasPriceProp).finish(),
-        },
-        newAmino: {
-          ...amino,
-        },
-      };
-    }
+    // TODO: add support for normal messages to run gov props via amino
```

Comment:
will this be supported in the near future or? can remove the code instead of comment?

Reply:
left it there for reference, i think can assign someone to add the amino support for gov props for those messages

Reply:
ok